tests/test_s3_specific.py

```diff
@@ -133,10 +137,11 @@ def _execute_on_subprocess_and_observe(use_threads):
         assert _execute_on_subprocess_and_observe(use_threads=True) > 10
 
 
-def test_no_sign_request(s3_rig):
+def test_no_sign_request(s3_rig, tmp_path):
     """Tests that we can pass no_sign_request to the S3Client and we will
     be able to access public resources but not private ones.
     """
+    tmp_path = Path(tmp_path)
```

Comment:
`tmp_path` is already a `Path` object. ([docs](https://docs.pytest.org/en/stable/tmpdir.html#the-tmp-path-fixture))